config/routes.rb

```diff
@@ -18,7 +18,7 @@
   namespace :insights_cloud do
     resources :tasks, only: [:create]
     resource :settings, only: [:show, :update]
-    resources :hits, except: %i[show] do
+    resources :hits, except: [:show, :edit, :destroy, :update, :create, :new] do
```

Comment:
should we switch to `only` ?

Reply:
Didn't try `only` with more definitions inside